homeworks/week6/Man_Mai

```diff
@@ -0,0 +1,11 @@
+// I find this problem is quite tricky
+// My approach is that we I try to compare the identify the string s that include all the characters in T. String that sastifies this condition is called Good String
+// Then I continuously compare the current Good String with previous one everytime I detect a new Good String and return the shortest Good String.
+// after Finishing for looking all the GoodString, I return the shortest one.
+
+//Based on the this approach, I expect the time complexity will be O(n^2)
```

Comment:
What you described is a naive solution. The optimal solution can be O(N), to be exact O(M+N) where M and N are size of input strings.